docs/architecture/types/property-path.md

```diff
@@ -0,0 +1,27 @@
+# Property Paths
+
+[`PropertyValue`s](TODO: document these) in Pulumi are often grouped together either as a map of `PropertyKey` to `PropertyValue`, or as a single object property value.
```

Comment:
Worth pointing out this is a `PropertyMap`

Reply:
Done :heavy_check_mark: